src/dmd/backend/outbuf.d

```diff
@@ -75,13 +65,22 @@ struct Outbuffer
         return ret;
     }
 
+    private extern(D) inout(ubyte)[] opSlice(size_t from, size_t to) inout
+        @trusted pure nothrow @nogc
+    {
+        assert(this.buf, "Attempt to dereference a null pointer");
+        assert(from < to, "First index must be smaller than the second one");
+        assert(this.length() <= (to - from), "Out of bound access");
+        return this.buf[from .. to];
+    }
+
```

Comment:
why the move of this? I assume it hasn't changed?

Reply:
I thought I'd put the public interface followed by the private stuff, but apparently putting `private` there was wrong in the first place.

Reply:
oh, I see it was made private. Totally missed that before. Good reason to keep diffs as small as possible and do moves separately.